src/pages/console/tutorial/extend-cli.mdx

```diff
@@ -22,7 +22,34 @@ import all1 from "/src/fragments/pull-cli-studio.mdx";
 
 <Fragments fragments={{all: all1}} />
 
-## Adding features with the Amplify CLI
+<Callout>
+
+For Safari browser, the browser will provide a CLI login key. Copy the CLI login key from the browser and paste it into your terminal.
+
+</Callout>
+
+## Use component in code
+
+Now that your components are synced, navigate to a component and choose **Configure**. To get your React code, choose **Get component code**.
+
+Follow the instructions in the modal. If this is your first time setting up the UI components in your app, make sure to complete **Initial project setup** first to install all dependencies.
```

Comment:
should "initial project setup" link to another doc or heading?

Reply:
added missing steps for initial setup